packages/wonder-blocks-core/components/with-ssr-placeholder.js

```diff
@@ -129,6 +129,31 @@ export default class WithSSRPlaceholder extends React.Component<Props, State> {
         const {children, placeholder} = this.props;
 
         switch (renderState) {
+            // There are edge cases where for some reason, we get an unknown
+            // context value here. So far it seems to be when we're nested in a
+            // v1 WithSSRPlaceholder equivalent component, or in some older
+            // React v16 situations where we're nested in the provider of a
+            // different context.
+            //
+            // We ignore this from coverage. It's a maintenance case to help
+            // us catch code changes that affect the control flow unexpectedly,
+            // but it's not something we need to write a test case for.
+            //
+            /* istanbul ignore next */
+            default:
```

Comment:
Because I felt it was worth highlighting as important given the perhaps unexpected behavior of it and that it falls through to the "root" case directly below it, so it makes for a more readable flow if it sits above it.